src/main/kotlin/gg/essential/elementa/components/Window.kt

```diff
@@ -19,7 +19,7 @@ import java.util.concurrent.TimeUnit
  * or animating.
  */
 class Window @JvmOverloads constructor(
-    private val version: ElementaVersion,
+    val version: ElementaVersion,
```

Comment:
Would keep it internal for now. Users shouldn't have any reason to care about it. We can always make it public at a later point if we need to, but we ever go back.